lib/karma-electron-launcher.js

```diff
@@ -1,11 +1,22 @@
 // Load in our dependencies
 // DEV: We use `require`/`assert` instead of `peerDependencies` to make floating dependencies easier
 var electronPrebuilt;
-try {
+
+function moduleExists(moduleName) {
+  try {
+    return require.resolve(moduleName);
+  } catch (err) {
+    return false;
+  }
```

Comment:
I went with a function-less set of try/catch-es to reduce the amount of redundant code and `require's`. Personal preference =/